.mergify.yml

```diff
@@ -79,21 +77,28 @@ pull_request_rules:
         message: Automatically approving mergify
   - name: automatic squash and merge with success checks and the files matching the regex ^.mergify.yml is modified.
     conditions:
-      - check-success=fleet-ci/pr-merge
-      - label=automation
+      - check-success=buildkite/elastic-agent
       - files~=^\.mergify\.yml$
       - head~=^add-backport-next.*
       - "#approved-reviews-by>=1"
     actions:
       queue:
         name: default
+  - name: delete upstream branch for mergify backports
+    conditions:
+      - or:
+        - merged
+        - closed
+      - and:
+        - head~=^mergify/bp/.*
+    actions:
+      delete_head_branch:
   - name: delete upstream branch with changes on ^.mergify.yml that has been merged or closed
     conditions:
       - or:
         - merged
         - closed
       - and:
-        - label=automation
```

Comment:
this label is not something used anymore, apparently -> https://github.com/elastic/elastic-agent/pull/6645